website/src/js/PlaygroundViewModel.ts

```diff
@@ -306,6 +306,8 @@ export class PlaygroundViewModel {
 		}
 
 		this.code('<?php declare(strict_types = 1);\n' +
+			'use function PHPStan\\dumpType;\n' +
+			'use function PHPStan\\Testing\\assertType;\n' +
 			'\n' +
 			'class HelloWorld\n' +
```

Comment:
also I would suggest:
- the class to be named "Foo"
- the method to named "test"
- add one `@param` above the method as typing the "/**" always take me some time

Reply:
Thanks for the suggestion, but that's outside the scope of my intention for this PR. Feel free to open your own PR though with the changes!